std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -499,3 +504,16 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
     assert(b.length == 100);
     assert((() nothrow @nogc => a.deallocate(b))());
 }
+
+@system shared unittest
+{
+    import std.experimental.allocator : processAllocator, RCISharedAllocator;
+
+    alias SharedAllocT = shared AffixAllocator!(RCISharedAllocator, int);
+    assert(is(RCISharedAllocator == shared));
```

Comment:
You should also add an assert which verifies that `SharedAllocT`'s methods are actually inferred as `shared`.

Reply:
I'm not sure how annotating the unittest with `shared` helps.